src/Pages/Auth/Login.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import { useFormik } from "formik";
+import { Link } from "react-router-dom";
+import * as Yup from "yup";
+import { TimelineLite, Power3 } from "gsap";
+import { makeAuthenticatedApiCall } from "../Api.js";
+
+const url = "https://atumaatu.herokuapp.com/v1/sessions/";
```

Comment:
Can you move this URL to the api.js file. It is consistent and not going to change. The only thing you will need to pass with be the `sessions` part. You can rename to route

```suggestion
const path = "sessions";
```

Reply:
Ohk